services/newsfeed/jobs/drupal.js

```diff
@@ -39,8 +39,10 @@ const reloadNewsFromDrupal = async drupalSources => {
 
 	drupalSources.forEach(async source => {
 		const response = await requestLib(source.url);
-		const normalizedData = normalizeData(source, response.data);
-		await newsfeedDB.upsert(normalizedData);
+		if (response && response.data) {
+			const normalizedData = normalizeData(source, response.data);
+			await newsfeedDB.upsert(normalizedData);
+		}
```

Comment:
Recommended to avoid merge conflicts from https://github.com/LiskHQ/lisk-service/pull/1456.
```suggestion
		const normalizedData = response ? normalizeData(source, response.data) : [];
		await newsfeedDB.upsert(normalizedData);
```